packages/@n8n/constants/src/index.ts

```diff
@@ -45,3 +45,9 @@ export const UNLIMITED_LICENSE_QUOTA = -1;
 
 export type BooleanLicenseFeature = (typeof LICENSE_FEATURES)[keyof typeof LICENSE_FEATURES];
 export type NumericLicenseFeature = (typeof LICENSE_QUOTAS)[keyof typeof LICENSE_QUOTAS];
+
+const { NODE_ENV } = process.env;
```

Comment:
isn't `@n8n/constants` meant to be shared between frontend and backend? if yes, we should maybe keep these in in `@n8n/backend-common`?